etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/OrderMock.groovy

```diff
@@ -10,25 +10,61 @@ class OrderMock<T> implements Order<T> {
     private String fhirResourceId
     private String patientId
     private T underlyingOrders
+    private String placerOrderNumber
+    private String sendingApplicationDetails
+    private String sendingFacilityDetails
+    private String receivingApplicationDetails
+    private String receivingFacilityDetails
 
-    OrderMock(String fhirResourceId, String patientId, T underlyingOrders) {
+    OrderMock(String fhirResourceId, String patientId, T underlyingOrders, String placerOrderNumber, String sendingApplicationId, String sendingFacilityId,
+    String receivingApplicationDetails, String receivingFacilityDetails) {
         this.fhirResourceId = fhirResourceId
         this.patientId = patientId
         this.underlyingOrders = underlyingOrders
+        this.placerOrderNumber = placerOrderNumber
+        this.sendingApplicationDetails = sendingApplicationDetails
+        this.sendingFacilityDetails = sendingFacilityDetails
+        this.receivingApplicationDetails = receivingApplicationDetails
+        this.receivingFacilityDetails = receivingFacilityDetails
     }
 
     @Override
     T getUnderlyingOrder() {
-        return underlyingOrders
+        return this.underlyingOrders
     }
 
     @Override
     String getFhirResourceId() {
-        return fhirResourceId
+        return this.fhirResourceId
     }
 
     @Override
     String getPatientId() {
         return patientId
     }
+
+    @Override
+    String getPlacerOrderNumber() {
+        return this.placerOrderNumber
```

Comment:
I've seen some places in the project where we don't use `this` explicitly. Do we have a convention for this?

Reply:
I have seen it as well. Personally, I like to use `this` as it adds a little more reference to the viewer. Maybe we can decide on a convention to follow for the project.